microsoftgraph/microsoft-graph-explorer-v2
# Task/responsive

## Overview

Makes Graph Explorer responsive

### Demo

N/A

### Notes

I opted to use `bootstrap grid` instead of `ms-Grid` because I could not find documentation for the latter.

## Testing Instructions

* Checkout this branch & launch GE
* Open dev tools
* Try different screen sizes
* Observe that the UI doesn't break
